kolibri_explore_plugin/assets/src/views/DevTag.vue

```diff
@@ -0,0 +1,75 @@
+<template>
+
+  <div v-if="dev" class="dev fixed-top mx-1 text-right">
+    <span v-b-modal.buildinfo class="badge badge-warning">{{ title }}</span>
+
+    <b-modal id="buildinfo" title="Build Info" okOnly>
+      <b-list-group>
+        <b-list-group-item>Commit: {{ info.commit }}</b-list-group-item>
+        <b-list-group-item>Date: {{ info.date }} - {{ info.time }}</b-list-group-item>
+      </b-list-group>
+      <h3>Log</h3>
+      <b-list-group>
+        <b-list-group-item v-for="commit in info.log" :key="commit.commit">
+          {{ commit.subject }} <br> <i>{{ commit.author }}</i>
+        </b-list-group-item>
+      </b-list-group>
+    </b-modal>
+  </div>
+
+</template>
+
+
+<script>
+
+  import urls from 'kolibri.urls';
+
+  export default {
+    name: 'DevTag',
+    data() {
+      return {
+        info: null,
+      };
+    },
+    computed: {
+      dev() {
```

Comment:
Please use better names. For booleans is always better to use a prefix like "isFoo", "hasBar", "canBaz". And avoid abbreviations. Maybe "isDevelopmentBuild"?

Reply:
I've used `showBuildInfo` that's more descriptive.